controls/srg_gpos/SRG-OS-000120-GPOS-00061.yml

```diff
@@ -9,8 +9,6 @@ controls:
             - kerberos_disable_no_keytab
             - accounts_password_all_shadowed_sha512
             - package_rsyslog-gnutls_installed
-            - rsyslog_remote_tls
-            - package_libreswan_installed
+            - libreswan_approved_tunnels
             - configure_kerberos_crypto_policy
-            - package_krb5-workstation_removed
```

Comment:
I see that in RHEL8 STIG this rule is still there with this SRG. What is the reason behind this change?

Reply:
Recent versions of the package appear to use FIPS-compliant cryptographic modules. See the latest
RHEL8 STIG (RHEL-08-010162) for more details.

Reply:
makes sense, maybe we will need to provide documentation that krb5 can use the FIPS stuff at some point

Reply:
Possibly, I was trying to find some docs for it. I think we need, in general, to find docs (or something) on the status of FIPS and krb5 in RHEL9. Do we want to merge this now and raise an issue to figure out what is going on?

Reply:
Let's merge this one now and I will create an issue to track this.